app/views/items/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for [user, item] do |f| %>
+  <div class="form-group">
+    <%= f.label :description %>
+    <%= f.text_field :description, class: 'form-control', placeholder: "Enter the task" %>
```

Comment:
I'm assuming this is where the bug is.

Take a look at your `schema.rb` file.  Notice that items don't have a column called "description".  So the problem is caused because you have a form input field for an attribute the the model doesn't know about.